WordPressShared/Core/Analytics/WPAnalytics.h

```diff
@@ -74,7 +74,6 @@ typedef NS_ENUM(NSUInteger, WPAnalyticsStat) {
     WPAnalyticsStatEditorAddedPhotoViaWPMediaLibrary,
     WPAnalyticsStatEditorAddedVideoViaWPMediaLibrary,
     WPAnalyticsStatEditorAddedOtherMediaViaWPMediaLibrary,
-    WPAnalyticsStatEditorAddedPhotoViaGiphy,
```

Comment:
Will the related Tenor tracks be added in a separate PR?

Reply:
Nope. The Tenor tracks are using the [new system](https://wp.me/paNNhX-55) that doesn't require Shared. And have [already been added](https://github.com/wordpress-mobile/WordPress-iOS/pull/13923) to WP.